### PR TITLE
fix for help menu not opening on mac #528

### PR DESCRIPTION
**Pull request by moldover**
Convert help.txt file resource URL to file _path_ and use full path to 'open' util does the trick for me.  Pretty sure this should work on other platforms, but worth checking when launching from the OS i.e. not using intelliJ.

### CODE REVIEW

File: src/main/java/network/brightspots/rcv/GuiConfigController.java [hunk 1]
```diff
@@ -307,20 +307,21 @@ private String getOvervoteRuleChoice() {
...
     } else if (Utils.IS_OS_MAC) {
-      command = String.format("open %s", helpFileUrl);
+      command = String.format("/usr/bin/open %s", helpFile);
     } else if (Utils.IS_OS_LINUX) {
-      command = String.format("xdg-open \"%s\"", helpFileUrl);
+      command = String.format("xdg-open \"%s\"", helpFile);
     } else {
       Logger.info("Unable to determine operating system. Try opening the documentation "
-          + "manually at: %s", helpFileUrl);
+          + "manually at: %s", helpFile);
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/GuiConfigController.java:320]: So this won't work on my Amiga?

File: src/main/java/network/brightspots/rcv/GuiConfigController.java [hunk 2]
```diff
@@ -307,20 +307,21 @@ private String getOvervoteRuleChoice() {
...
     }
     if (command != null) {
       try {
+        Logger.info("Running: %s", command);
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/GuiConfigController.java:324]: I don't think we need this to be logged.

**Review by tarheel — APPROVED** [review]

**Comment by tarheel** [comment]
LGTM, but will let @HEdingfield weigh in also.

**Review by HEdingfield — APPROVED** [review]

File: src/main/java/network/brightspots/rcv/GuiConfigController.java (hunk 2 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/GuiConfigController.java]: Ok, yeah this is removed.

File: src/main/java/network/brightspots/rcv/GuiConfigController.java (hunk 1 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/GuiConfigController.java]: No Newton either :/

Comment (tarheel) [reply src/main/java/network/brightspots/rcv/GuiConfigController.java]: https://www.youtube.com/watch?v=u6qxixgQJ4M

**Pull request merged**